websites/A/AnizmTV/presence.ts

```diff
@@ -78,9 +78,12 @@ presence.on("UpdateData", async () => {
 	}
 
 	//Episode part
+	let episodePresence;
 	if (title && episode) {
 		presenceData.details = title.textContent;
-		presenceData.state = episode.textContent.split("/ ").slice(1).join(" ");
+		episodePresence = episode.textContent.split("/ ").slice(1).join(" ");
+		episodePresence = episodePresence + " İzliyor";
+		presenceData.state = episodePresence;
```

Comment:
Is this all you need to change in the code to support all of those URLs?

Reply:
The code I've included here is for "watching episode 10". Previously, it only read "Episode 10". Bas1c hadn't added it before, it caught my eye while I was dealing with it today and I added it.

Reply:
so you're telling me all those URLs send you to the exact same website and the presence works as-is on all?

Reply:
Yes, all websites use the same server and the same database. There is no difference from each other in terms of content and design.